Client/Card.ts

```diff
@@ -19,10 +19,52 @@ export class Card {
 	async get(card: authly.Token): Promise<ModelCard | gracely.Error> {
 		return await this.connection.get<ModelCard | gracely.Error>(`card/${card}`)
 	}
+	async proxy(
+		method: "post" | "get" | "patch" | "put" | "delete",
+		config: string,
+		body: any,
+		path: string,
+		token: string,
+		selectors: Record<string, any>,
+		proxyAuthorization: string
+	): Promise<any> {
+		let result: any
+		if (method == "get" || method == "delete")
+			result = this.connection[method](`card/proxy/${config}/${path}`, { "Proxy-Authorization": proxyAuthorization })
+		else
+			result = this.connection[method](`card/proxy/${config}/${path}`, createBody(body, selectors, token), {
+				"Proxy-Authorization": proxyAuthorization,
+			})
+		return await result
+	}
+
 	static open(url: string, key: string): Card
 	static open(url?: string, key?: string): Card | undefined
 	static open(url?: string, key?: string): Card | undefined {
 		const connection = Connection.open(url, key)
 		return connection && new Card(connection)
 	}
 }
+
+export function createBody(
+	body: Record<string, any>,
+	selectors: Record<string, any>,
+	token: string
+): Record<string, any> {
+	let result: Record<string, any>
+	if (!selectors)
+		result = body
+	else {
+		Object.entries(selectors).forEach(([key, value]) => {
+			const target = body[key]
+			body[key] =
+				!(Array.isArray(target) && Array.isArray(value)) && typeof target == "object" && typeof value == "object"
+					? createBody(Object.assign({}, target), value, token)
```

Comment:
I think this is exactly the same:
```suggestion
					? createBody({ ...target }, value, token)
```